src/cascadia/TerminalSettingsModel/TerminalWarnings.idl

```diff
@@ -8,20 +8,19 @@ namespace Microsoft.Terminal.Settings.Model
     enum SettingsLoadWarnings
     {
         MissingDefaultProfile = 0,
-        DuplicateProfile = 1,
-        UnknownColorScheme = 2,
-        InvalidBackgroundImage = 3,
-        InvalidIcon = 4,
-        AtLeastOneKeybindingWarning = 5,
-        TooManyKeysForChord = 6,
-        MissingRequiredParameter = 7,
-        LegacyGlobalsProperty = 8,

```

Comment:
Yeah I removed the code responsible for it. Supporting this would've required me to add an extra check in `SettingsLoader::_parse` and wire up the warning propagation. I've decided that it's been long enough since the warning got introduced and thus removed it.